src/modules/site-v2/templates/tools/variant_annotation/vbrowser.html

```diff
@@ -261,6 +262,17 @@
 // Track changes to selected species
 function update_species(species_id) {
 
+  // Enable or disable form inputs that depend on species being defined
+  const form_ids = [
+    'gene-search',
+    'interval',
+    'search-interval-btn',
+    'strain-filter',
+  ]
+  form_ids.forEach((form_id) => {
+    document.getElementById(form_id).disabled = !species_id;

```

Comment:
this is legacy code, but can we clarify the semantics of `!species_id`?

Reply:
Fixed in 0ef91f28589e0a20241e7923fa789b5da1f35672